docs/usage.md

```diff
@@ -3,130 +3,317 @@
+
+    >*Access the full details of the alert for further analysis*
+
+   - The assistant displays the comprehensive information related to the opened record.
+
+### Review Context Information
+
+- Review the additional information provided to ensure it meets your needs. Verify that the context is accurate and relevant to the current task or analysis.
+
+### Integrate Context into Workflow
+
+- Incorporate the provided context into your workflow to enhance decision-making and task management. Utilize the additional information to inform your actions and streamline processes.
+
+Providing context about an *already opened* record is essential for effective analysis and decision-making within FortiSOAR. By following the steps outlined and leveraging the FortiAI assistant’s capabilities, users can access and utilize relevant information to improve their workflows.
+
+>[!NOTE]
+>Ensure that context provided is relevant and up-to-date to maintain accuracy in decision-making and task execution.
+
+## Link Record
+
+Linking records within the FortiAI assistant involves correlating a record with other relevant records. This functionality helps establish connections between different pieces of information, such as correlating an indicator with an existing alert, to enhance data analysis and response actions.
+
+### Use Case Example: Correlating an Indicator
+
+A common use case is correlating an indicator with an existing alert. For example, if an alert is triggered due to a specific indicator, linking the indicator to the alert helps in understanding the relationship and impact. This correlation can aid in more accurate threat analysis and response.
+
+### Linking Records Using FortiAI Assistant
 
-### Generate playbook steps using FortiAI
+1. **Initiate Linking**:
+   - Issue a command to link a record with another relevant record. For example:
 
-Assume for the moment that you are a playbook designer who is required to build workflows to mitigate various types of threats. 
+    >*Link indicator ID 67890 to alert ID 12345*
 
-  >**NOTE**: It is recommended to create smaller playbook blocks for easier review and minimal changes in the JSON template generated by FortiAI. You can then combine these small playbook blocks to create a larger playbook for your desired workflow.
+   - The assistant establishes the connection between the specified indicator and alert.
 
-One use case to mitigate threats could require playbook steps that that extract indicators from an alert and enrich them using a threat intelligence tool such as VirusTotal.
+2. **Correlate Records**:
+   - Specify the records you want to correlate. For example:
 
-Based on the reputation, the indicator reputation is updated to *Malicious*, if the reputation returned by VirusTotal is *Suspicious*; or the indicator reputation is updated to *Good* if the reputation is anything but *Suspicious*.
+    >*Correlate the newly detected IP address with existing incidents*
```

Comment:
Change this prompt to "Correlate the indicator id 1 address with this incidents"

Reply:
Done